internal/primitives/core/hashing/hashing.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package hashing
+
+import (
+	"golang.org/x/crypto/blake2b"
+)
+
+// BlakeTwo256 returns a Blake2 256-bit hash of the input data
+func BlakeTwo256(data []byte) [32]byte {
```

Comment:
If we change `type H256 string` to `type H256 [32]byte` we could change this to:

```suggestion
func BlakeTwo256(data []byte) H256 {
```

And I think it will make totally sense

Reply:
I think in the context of this package I think it makes sense to return `[32]byte`.